Model/src/main/java/org/gusdb/wdk/model/dbms/ResultFactory.java

```diff
@@ -162,7 +162,7 @@ private void insertInstanceRow(InstanceInfo instanceInfo) throws WdkModelExcepti
         Types.VARCHAR, // table name
         Types.VARCHAR, // query name
         Types.VARCHAR, // checksum
-        Types.CLOB     // result message
+        Types.OTHER     // result message
```

Comment:
Curious about what led to this choice.  Can we use [platform functions](https://github.com/VEuPathDB/FgpUtil/blob/f0d54249aaf49c0486b018e2811e5ef3864fbd59/Db/src/main/java/org/gusdb/fgputil/db/platform/PostgreSQL.java#L65) to do this?  Looks like we just have schema gen and writing methods, but I think that's because writing the value can be [the same on both DBs](https://github.com/VEuPathDB/FgpUtil/blob/f0d54249aaf49c0486b018e2811e5ef3864fbd59/Db/src/main/java/org/gusdb/fgputil/db/platform/DBPlatform.java#L173)?  Maybe with a different column type in postgres?